spec/RequestDispatcher_spec.lua

```diff
@@ -54,12 +54,32 @@ describe("RequestDispatcher", function()
         adapter_mock.create_virtual_schema = function(_, _, properties)
             recorded_properties = properties
         end
-        local dispatcher_with_adapter_mock = RequestDispatcher:new(adapter_mock)
+        local dispatcher_with_adapter_mock = RequestDispatcher:new(adapter_mock, AdapterProperties)
         local raw_request = '{"type" : "createVirtualSchema", "schemaMetadataInfo" : {"properties" : {"FOO" : "bar"}}}'
         dispatcher_with_adapter_mock:adapter_call(raw_request)
         assert.is.equal("bar", recorded_properties:get("FOO"))
     end)
 
+    it("dispatches property setting [utest -> dsn~dispatching-set-properties-requests~0]", function()
```

Comment:
We usually add coverage as comment, but this is also nice since it can be written in the test name.

Reply:
But why is it ends with `~0`? I though we start the revisions with 1 usually.

Reply:
That is due to a still missing feature when using requirement forwarding. At the moment we only support the following forwarding in the tag importer:

`[dsn -> req~foo~1 >> utest, itest]`, which assigns revision zero to the intermediate spec item `dsn~foo~0`.

What we actually want is to copy the revision. Or even better:

`[dsn~~2 -> req~foo~1 >> utest, itest]` &rarr; `dsn~foo~2`.